data_transfer/dags/btf.py

```diff
@@ -1,13 +1,21 @@
+import logging
 from datetime import datetime
+from typing import Union
 
-from data_transfer.db import records_not_uploaded
+from data_transfer.config import config
+from data_transfer.db import all_records_downloaded, records_not_uploaded
+from data_transfer.devices.byteflies import Byteflies
 from data_transfer.jobs import byteflies as byteflies_jobs
 from data_transfer.jobs import shared as shared_jobs
 from data_transfer.tasks import byteflies as byteflies_tasks
 from data_transfer.utils import DeviceType, StudySite, get_period_by_days
 
+log = logging.getLogger(__name__)
 
-def dag(study_site: StudySite) -> None:
+
+def dag(
+    study_site: StudySite, days: Union[int, str] = 50, delta: Union[int, str] = 0
+) -> None:
```

Comment:
I'd make the parameters explict types (ints) as these are defined once in the CLI (main.py) and can be casted there once rather than below and elsewhere. The docstring also indicates these are ints, but that they are optional which isn't quite correct?

Reply:
I'm casting here because due to our CLI use of this library, `delta` and `days` might be string